### PR TITLE
Clean up review comments from #1929

### PR DESCRIPTION
**Pull request by jeanphilippeD**

### CODE REVIEW

**Comment by jeanphilippeD** [comment]
Rebased.

File: src/states/elder/mod.rs [hunk 1]
```diff
@@ -60,7 +60,10 @@ use crate::messages::Message;
 
 /// Time after which a `Ticked` event is sent.
 const TICK_TIMEOUT: Duration = Duration::from_secs(15);
+/// Time after which an Elder should send a new Gossip.
 const GOSSIP_TIMEOUT: Duration = Duration::from_secs(2);
+/// Number of RelocatePrepare to consensus before actually relocating a node.
+/// This helps avoid relocated node receiving message they need to prcess from previous section.
```
Comment (madadam) [inline src/states/elder/mod.rs]: Typo: "...they need to pr**o**cess..."

File: src/states/adult.rs [hunk 2]
```diff
@@ -306,19 +306,20 @@ impl Adult {
...
+    // further guide the joining node.
+    // However this lead to a loop if the Adult is the new Elder so we use the same code as
+    // in Elder and return Join in some cases.
     fn handle_bootstrap_request(&mut self, p2p_node: P2pNode, destination: XorName) {
-        self.respond_to_bootstrap_request(&p2p_node, &destination);
-    }
-
-    fn respond_to_bootstrap_request(&mut self, p2p_node: &P2pNode, name: &XorName) {
-        let response = if self.our_prefix().matches(name) {
+        // Use asme code as from Elder::respond_to_bootstrap_request.
```
Comment (madadam) [inline src/states/adult.rs]: Type: "Use same..."

File: src/states/adult.rs [hunk 3]
```diff
@@ -306,19 +306,20 @@ impl Adult {
         }
     }
 
-    // Since we are an adult we will only give info about our section elders and they will further
-    // guide the joining node.
+    // Since we are an adult we should only give info about our section elders and they would
+    // further guide the joining node.
+    // However this lead to a loop if the Adult is the new Elder so we use the same code as
+    // in Elder and return Join in some cases.
```
Comment (madadam) [inline src/states/adult.rs]: Should we create a github issue for this?

Comment (jeanphilippeD) [reply src/states/adult.rs]: https://github.com/maidsafe/routing/issues/1962

**Comment by jeanphilippeD** [comment]
Addressed review comments

**Pull request merged**